test/integration/content-sidebar/ContentSidebar.e2e.test.js

```diff
@@ -44,6 +44,26 @@ describe('ContentSidebar', () => {
             cy.getByTestId('sidebarskills').should('not.have.class', 'bcs-is-selected');
         });
 
+        it('should switch to the next panel when a user presses down arrow', () => {
+            cy.getByTestId('bcs-content').should('exist');
+            cy.getByTestId('sidebarskills').should('have.class', 'bcs-is-selected');
+
+            cy.getByTestId('sidebarskills').trigger('keydown', { key: 'ArrowDown' });
+            cy.getByTestId('sidebarskills').should('not.have.class', 'bcs-is-selected');
+            cy.getByTestId('sidebarmetadata').should('have.class', 'bcs-is-selected');
+            cy.focused().should('have.attr', 'data-testid', 'sidebarmetadata');
+        });
+
+        it('should switch to the previous panel when a user presses up arrow ', () => {
```

Comment:
These can be combined into a single test to save on the overhead of starting a new test case.